doc/api.rst

```diff
@@ -590,6 +590,298 @@ Reshaping and reorganizing
    DataArray.sortby
    DataArray.broadcast_like
 
+DataTree
+========
+
+Creating a DataTree
+-------------------
+
+Methods of creating a ``DataTree``.
+
+.. autosummary::
+   :toctree: generated/
+
+   DataTree
+   DataTree.from_dict
+
+Tree Attributes
+---------------
+
+Attributes relating to the recursive tree-like structure of a ``DataTree``.
+
+.. autosummary::
+   :toctree: generated/
+
+   DataTree.parent
+   DataTree.children
+   DataTree.name
+   DataTree.path
+   DataTree.root
+   DataTree.is_root
+   DataTree.is_leaf
+   DataTree.leaves
+   DataTree.level
+   DataTree.depth
+   DataTree.width
+   DataTree.subtree
+   DataTree.descendants
+   DataTree.siblings
+   DataTree.lineage
+   DataTree.parents
+   DataTree.ancestors
+   DataTree.groups
+
+Data Contents
+-------------
+
+Interface to the data objects (optionally) stored inside a single ``DataTree`` node.
+This interface echoes that of ``xarray.Dataset``.
+
+.. autosummary::
+   :toctree: generated/
+
+   DataTree.dims
+   DataTree.sizes
+   DataTree.data_vars
+   DataTree.coords
+   DataTree.attrs
+   DataTree.encoding
+   DataTree.indexes
+   DataTree.nbytes
+   DataTree.ds
+   DataTree.to_dataset
+   DataTree.has_data
+   DataTree.has_attrs
+   DataTree.is_empty
+   DataTree.is_hollow
+
+Dictionary Interface
+--------------------
+
+``DataTree`` objects also have a dict-like interface mapping keys to either ``xarray.DataArray``s or to child ``DataTree`` nodes.
```

Comment:
The s together to the backticks breaks the rendering, it should either be removed or have a space between the backticks and the s. https://xray--9033.org.readthedocs.build/en/9033/api.html#id9